CHANGELOG.md

```diff
@@ -28,6 +28,33 @@ API after
 client = substra.Client(backend_type=substra.BackendType.LOCAL_SUBPROCESS)
 ```
 
+- BREAKING CHANGE: the substratools `Metrics` is now another algo (#290):
```

Comment:
this is not a substra breaking change. You could mention that substra test has been adapted due to breaking changes within substra tools